projects/control-service/projects/helm_charts/pipelines-control-service/templates/_helpers.tpl

```diff
@@ -133,54 +133,6 @@ Also, we can't use a single if because lazy evaluation is not an option
 {{- end -}}
 {{- end -}}
 
-{{/*
-Return the proper Control Service data jobs base image where data job run in.
-*/}}
-{{- define "pipelines-control-service.deploymentDataJobBaseImage" -}}
-{{- $globalRegistryOverrideName := .Values.deploymentDataJobBaseImage.globalRegistryOverride -}}
-{{- $registryName := .Values.deploymentDataJobBaseImage.registry -}}
-{{- $repositoryName := .Values.deploymentDataJobBaseImage.repository -}}
-{{- $tag := .Values.deploymentDataJobBaseImage.tag | toString -}}
-{{/*
-Helm 2.11 supports the assignment of a value to a variable defined in a different scope,
-but Helm 2.9 and 2.10 doesn't support it, so we need to implement this if-else logic.
-Also, we can't use a single if because lazy evaluation is not an option
-*/}}
-{{- if $globalRegistryOverrideName }}
-    {{- printf "%s/%s:%s" $globalRegistryOverrideName $repositoryName $tag -}}
-{{- else if .Values.global.imageRegistry }}
-    {{- printf "%s/%s:%s" .Values.global.imageRegistry $repositoryName $tag -}}
-{{- else -}}
-    {{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
-{{- end -}}
-{{- end -}}
-
-{{/*
-Return the proper Control Service vdk image where the vdk (sdk) is installed and used during data job runs.
-*/}}
-{{- define "pipelines-control-service.deploymentVdkDistributionImageRepository" -}}
-    {{- $globalRegistryOverrideName := .Values.deploymentVdkDistributionImage.globalRegistryOverride -}}
-    {{- $registryName := .Values.deploymentVdkDistributionImage.registry -}}
-    {{- $repositoryName := .Values.deploymentVdkDistributionImage.repository -}}
-
-    {{/*
-    Helm 2.11 supports the assignment of a value to a variable defined in a different scope,
-    but Helm 2.9 and 2.10 doesn't support it, so we need to implement this if-else logic.
-    Also, we can't use a single if because lazy evaluation is not an option
-    */}}
-    {{- if $globalRegistryOverrideName }}
-        {{- printf "%s/%s" $globalRegistryOverrideName $repositoryName -}}
-    {{- else if .Values.global.imageRegistry }}
-        {{- printf "%s/%s" .Values.global.imageRegistry $repositoryName -}}
-    {{- else -}}
-        {{- printf "%s/%s" $registryName $repositoryName -}}
-    {{- end -}}
-{{- end -}}
-
-{{- define "pipelines-control-service.deploymentVdkDistributionImage" -}}
```

Comment:
nit: do we actually housekeep a documented nomenclature of those configs we need to update, or not at all?

Reply:
I don't think we do that, or at least I haven't seen a document discussing config nomenclature.

Reply:
We generally follow Bitnami guidelines for building our helm chart (https://docs.bitnami.com/tutorials/series/best-practices-helm) . If no explicit guidelines are given by Bitnami we should infer those from their helm chart (https://github.com/bitnami/charts). 

This is because we would like to integrate with Bitnami at some future point so we want to make this integration as easy as possible and reuse both practice, charts, ideas from [bitnami charts repository](https://github.com/bitnami/charts)